src/server/api.ts

```diff
@@ -82,7 +82,7 @@ export const addApiToServer = ({ app, db, games, lobbyConfig }) => {
 
   router.post('/games/:name/create', koaBody(), async ctx => {
     // The name of the game (for example: tic-tac-toe).
-    const gameName = ctx.params.name;
+    const gameName = ctx.params.name.replace(' ', '-');
```

Comment:
Hmm, I think this might open a whole other can of worms (if we start to allow malformed names in the API).